fix: OPTIC-7: Sum lead time from previous draft on annotation submission

robot-ci-heartex opened the pull request:
Hi @Travis1282!

This PR was created in response to a PRs in upstream repo:
- https://github.com/HumanSignal/dm2/pull/223
- https://github.com/HumanSignal/label-studio-frontend/pull/1542

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5904845701

Travis1282 committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5904845701

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5904898669

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5952861928

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-7/draft-lead-time-added'

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5952957190

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5952972739

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5954230125

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-7/draft-lead-time-added'

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5954511303

hlomzik commented:
/git merge develop
> Successfully pushed new changes:
> Merge remote-tracking branch 'origin/develop' into fb-optic-7/draft-lead-time-added (b1c4121794ad92fbb779857a4ac2630cab5ce4c8)
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5969200718)

hlomzik committed:
Merge remote-tracking branch 'origin/develop' into fb-optic-7/draft-lead-time-added

hlomzik commented:
/frontend build lsf master
> **Error**: failed to get build
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5969263636)

hlomzik commented:
/frontend build editor master
> **Error**: failed to get build
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5969270309)

hlomzik commented:
/frontend build lsf master


> **Error**: failed to get build
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5969288296)

hlomzik commented:
/frontend build lsf master


> **Error**: failed to get build
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5969576394)

hlomzik commented:
/frontend build lsf master


> **Error**: failed to get build
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5969593726)

hlomzik commented:
/frontend build lsf 3f8c90df983b9929c19115279306bac7ec272276
> **Error**: failed to get build
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5969612879)

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5969660495

hlomzik committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5969660495

hlomzik committed:
Merge branch 'origin/develop' into 'fb-optic-7/draft-lead-time-added'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5969660495

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6020388517

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-7/draft-lead-time-added'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6020388517

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6045032087

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-7/draft-lead-time-added'

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6113156723

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-7/draft-lead-time-added'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6113156723

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6151442388

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-7/draft-lead-time-added'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6151442388

robot-ci-heartex commented:
To enable Auto Merge for this PR also merge those PRs:
- https://github.com/HumanSignal/label-studio-frontend/pull/1542
